.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,26 @@
+name: PR Check
+on:
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      REF:
+        required: false
+        default: master
```

Comment:
Just curious, does this do anything?

Reply:
it is for triggering on a branch without PR, isn't it?

Reply:
it is to be able to run it agains any commit/branch/tag isn't it without a PR

Reply:
Ok I don't see it being used explicitly in the checkout action. Is this being used implicitly?